Update system test `example_dms_serverless` (#46684)
mistercrunch/Airflow: providers/amazon/docs/changelog.rst

* ``The experimental AWS auth manager is no longer compatible with Airflow 2``

Bug Fixes
~~~~~~~~~

> * ``The DMS waiter replication_terminal_status has been extended and wait for 2 additional status: "created" and "deprovisioned"``

Reviewer: ```suggestion
* ``The DMS waiter replication_terminal_status has been extended to proceed on 2 additional states: "created" and "deprovisioned"``
```

Author: Very good catch!
